public/v4/apps/matomo.yml

```diff
@@ -8,11 +8,11 @@ services:
             MYSQL_DATABASE: matomo
             MYSQL_USER: matomo
             MYSQL_PASSWORD: $$cap_db_pass
-            MYSQL_ROOT_PASSWORD: $$cap_db_root_pass
```

Comment:
Why is this removed? Isn't it needed for mysqldump?
To make it easier for the user, we can use `$$cap_gen_random_hex(18)` as the default value.

Reply:
I think you can just use the matomo users (who already have all the right on the table on the database)
I removed the usage of the root account to prevent missuse of root account but I can put it back if you prefear ?

Reply:
Ps : I'm not sure about the fact the user can use mysql_dump, I'll check the matomo documentation for dump

Reply:
I see. I didn't know there is an existing user that has access to all the tables. That should be fine then.